admin/tabs/server/logViewer/LogViewerModel.ts

```diff
@@ -72,11 +72,19 @@ export class LogViewerModel extends HoistModel {
         const {enabled, filesGridModel} = this;
         if (!enabled) return;
 
-        const store = filesGridModel.store as UrlStore,
+        const store = filesGridModel.store,
             selModel = filesGridModel.selModel;
 
+        // Get the data from 'listFileInfo`.
+        const data = await XH.fetchJson({
+            url: 'logViewerAdmin/listFileInfo'
+        });
+
+        const {logRootPath} = data;
```

Comment:
this variable here is just noise, used once on line below.

Reply:
also , it just occurs to me that we would like to just keep the `listFiles` endpoint in hoist-core with the same name for backward compatibility.  (realize we had talked about changing the name for clarity).  If there is no `logRootPath` key present (i.e. for an earlier version of Hoist Core), let's just quietly display nothing.

Reply:
Sounds good to me. At what point would a similar method's name get changed to avoid backward compatibility issues?